src/transformStateWithClones.js

```diff
@@ -1,13 +1,35 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} copy
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const res = [];
```

Comment:
Please, use more descriptive names for constants. Maybe here you can use something like 'arrayResults' or 'actionsResults'